src/pages/instances/InstanceDetailPanelContent.tsx

```diff
@@ -85,6 +89,14 @@ const InstanceDetailPanelContent: FC<Props> = ({ instance }) => {
             {settings?.environment?.server_clustered ? instance.location : "-"}
           </td>
         </tr>
+        <tr>
+          <th className="u-text--muted">Pool</th>
```

Comment:
I think we need to be more specific
```suggestion
          <th className="u-text--muted">Root storage pool</th>
```